### PR TITLE
Touchscreen compatible

### PR DESCRIPTION
**Pull request by NicholasLangley**
Now works with double taps (<= 500 ms apart) on the touchscreen. Not sure if wiringPi is actually used at all in the code or not

### CODE REVIEW

**Comment by bill-luu** [comment]
Continues #5

File: src/backup_camera.h [hunk 1]
```diff
@@ -22,7 +22,7 @@ extern "C" {
 #ifndef RUNNINGONPI
 #define RUNNINGONPI
 #endif
-#include "WiringPiButtons.hpp"
+#include <wiringPi.h>
```
Comment (enochtsang) [inline src/backup_camera.h]: Have you tried deleting this?

**Comment by enochtsang** [comment]
Has this been tested?

**Comment by bill-luu** [comment]
Yes, works great!

File: src/backup_camera.h (hunk 1 above)
Comment (enochtsang) [inline src/backup_camera.h]: Uhh I guess I merged a bit early, please look into this still. wiring pi was when we had GPIOs connected to the pi, but we don't need that anymore. Pretty sure we can delete all traces of it.

**Pull request merged**